doc/contribute/release_ccf.rst

```diff
@@ -24,4 +24,17 @@ Create an LTS release
 Create a dev release
 ---------------------
 
-    1. Tag the head of ``main`` as ``ccf-N+1.0.0-devX+1``, where ``N`` is the latest LTS, and ``X`` the latest dev release. 
\ No newline at end of file
+    1. Tag the head of ``main`` as ``ccf-N+1.0.0-devX+1``, where ``N`` is the latest LTS, and ``X`` the latest dev release.
+
+Labelling LTS PRs
+-----------------
+
+To keep track of which changes should be merged to LTS branches and then confirm them before cutting LTS releases, the following policy should be used:
+
+    1. PRs targetting ``main`` which contain changes that should also reach the LTS should be labelled in Github with ``lts-candidate``
+    2. Where possible ``lts-candidate`` PRs should contain the minimal set of changes with no dependencies on earlier PRs, so they can be more easily cherry-picked
+    3. PRs targetting an LTS branch (``release/N.x``) should be given labelled in Github with ``lts``, so they can be easily found
+    4. PRs targetting an LTS branch which are constructed primarily of a cherry-pick from ``main`` should be named consistently as ``Cherry-pick #<PR_ID> to N.x LTS``, so they can be easily correlated with the corresponding ``lts-candidate`` PRs
```

Comment:
I think we generalise that slightly to "A PR targetting an LTS branch should mention all PRs it contains commits from in its title, eg. "Backport feature foo to LTS (#123, #234, #456)".